webClient/src/app/shared/editor-control/editor-control.service.ts

```diff
@@ -456,6 +456,7 @@ export class EditorControlService implements ZLUX.IEditor, ZLUX.IEditorMultiBuff
        */
       this.snackBar.open(`${error}`, 'Close', { duration: 2000, panelClass: 'center' });
     });
+    this.openDirectory.next(results.directory);
```

Comment:
Because this is outside of the request to save the file, on a slow connection this will simply not work: the call can complete before the file you made is saved, so no update will be seen.
This needs to be moved inside of the save request, after you get a response whether success or error.